Sources/MapboxDirections/Directions.swift

```diff
@@ -158,48 +158,55 @@ open class Directions: NSObject {
         let requestTask = URLSession.shared.dataTask(with: request) { (possibleData, possibleResponse, possibleError) in
             let responseEndDate = Date()
             guard let response = possibleResponse, ["application/json", "text/html"].contains(response.mimeType) else {
-                completionHandler(nil, nil, .invalidResponse)
+                let response = RouteResponse(code: nil, message: nil, error: .invalidResponse(possibleResponse), uuid: nil, routes: nil, waypoints: nil)
```

Comment:
Let’s provide default argument values or an alternative initializer for the error case to reduce repetition.